src/diffusers/configuration_utils.py

```diff
@@ -378,6 +387,9 @@ def load_config(
         try:
             # Load config dict
             config_dict = cls._dict_from_json_file(config_file)
+
+            commit_hash = extract_commit_hash(config_file)
+            config_dict["_commit_hash"] = commit_hash
```

Comment:
Nice! I think in the future it would be nice if we had an option for the hub library to return additional metadata (including the commit hash) when we download a file. It looks like in the source, the commit hash is manually read out of the refs folder or the first metadata request from the hub. Relying on the structure of the local cache is ok but if we already retrieve the information in the function, would be nice to just explicitly return it. 

Sorry if I already said something similar to this, can't remember if I did :P